src/test/infrastructure/service/IABConsentHelperTest.js

```diff
@@ -0,0 +1,40 @@
+import {iabGenerateConsent} from '../../testable/infrastructure/consent/IABConsentUtils'
+import {IABConsentDecoderService} from '../../../main/infrastructure/service/IABConsentDecoderService'
+import {COOKIE} from '../../fixtures/cookie'
+import {TestableGVLFactory} from '../../testable/infrastructure/repository/iab/TestableGVLFactory'
+
+describe.skip('IABConsentHelper', () => {
+  it.skip('generate a consent', async () => {
+    console.log('>>> generating a consent')
+
+    const givenVersionToEncodeConsent = 36
+    const givenAcceptance = true
+
+    const gvlFactory = new TestableGVLFactory({
+      latestGvlVersion: givenVersionToEncodeConsent
+    })
+    const result = await iabGenerateConsent({
+      gvlFactory,
+      allPurposeConsents: givenAcceptance,
+      allPurposeLegitimateInterests: givenAcceptance,
+      allSpecialFeatures: givenAcceptance,
+      allVendorConsents: givenAcceptance,
+      allVendorLegitimateInterests: givenAcceptance,
+      editedVendorConsents: true,
+      editedVendorLegitimateInterests: false
+    })
+    // console.log('iabGenerateConsent', result)
```

Comment:
do you want to keep this console.log commented?

Reply:
It has been useful to me (to know that output vendors where not all the vendorlist vendors p.ex.) but I can remove